imports/api/resolvers.js

```diff
@@ -0,0 +1,26 @@
+import { Lists } from './lists/lists';
+
+export default {
+  RootQuery: {
+    lists(_, { mine = false }, context) {
+      if (mine) {
+        // XXX: is this the correct way to do auth?
```

Comment:
Oops nvm, I just meant trusting `context.userId`. I've also wondered what to do about errors